engine/src/main/java/org/destinationsol/world/generators/SolarSystemGenerator.java

```diff
@@ -382,6 +393,21 @@ public void buildFeatureGenerators() {
         }
     }
 
+    /**
+     * This method returns the Planet objects built by the PlanetGenerators of this SolarSystem. It is most useful
+     * after the build method is called
+     * @return List of Planets
+     */
+    ArrayList<Planet> getBuiltPlanets() {
```

Comment:
"Built" sounds funny to me here. "Generated" ? What exact state are the planets in before, after, and what happens next? Is surface feature placement and other stuff still to do? This is where again a diagram or two would be awesome to show the phases / state changes :-)